### ISSUE TITLE
Adding "Select all metrics" option

### ISSUE THREAD

**Issue by Khazhak** [aimhubio/aim#444]
**Problem**
When you have multiple metrics, it is annoying to go one-by-one and select to show them by hand. An option to select all available metrics would come in handy.

**Comment by vwxyzjn** [comment]
Yeah we could have hundreds of metrics, and wandb can show all of them by default. 
<img width="1859" alt="image" src="https://github.com/aimhubio/aim/assets/5555347/bf61905d-0df0-4a05-b459-54aeda3784f8">
